### PR TITLE
Let distributor metric decide cluster stability

### PR DESCRIPTION
**Pull request by mpolden**
@bratseth

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java [hunk 1]
```diff
@@ -107,7 +108,10 @@ private void consumeMetricsItem(Inspector item, ListMap<String, Double> values)
         item.field("values").traverse((ObjectTraverser)(name, value) -> values.put(name, value.asDouble()));
     }
 
-    private boolean clusterIsStable(Node node, NodeList applicationNodes) {
+    private boolean clusterIsStable(Node node, NodeList applicationNodes, ListMap<String, Double> nodeValues) {
+        if (Metric.redistributing.from(nodeValues) > 0) {
```
Comment (bratseth) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:112]: We should probably rely only on this?

**Review by bratseth — APPROVED** [review]
We can probably remove the previous method - otherwise LGTM!

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java (hunk 1 above)
Comment (mpolden) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:112]: Yes, but I don't think the merge pending metric will be exposed in the "autoscale" set until the application upgrades?

Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:112]: Good point!

**Pull request merged**